packages/eslint-plugin-pf-codemods/lib/rules/v5/emptyState-rename-components.js

```diff
@@ -0,0 +1,86 @@
+const { ensureImports, getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8737
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
+    const pfPackage = "@patternfly/react-core";
+    const oldNames = ["EmptyStatePrimary", "EmptyStateSecondaryActions"];
+
+    const imports = getPackageImports(context, pfPackage).filter((specifier) =>
+      oldNames.includes(specifier.imported.name)
+    );
+
+    const importNames = Object.fromEntries(
+      imports.map((imp) => [imp.local.name, imp.imported.name])
+    );
+
+    const newName = "EmptyStateActions";
+
+    return imports.length == 0
+      ? {}
+      : {
+          ImportDeclaration(node) {
```

Comment:
I don't think I'd block over it since we could do a followup if needed, but I'm wondering if we could simplify the fixer for the ImportDeclaration so that we don't have to have a test file that needs multiple runs to remove components then add the new one. I can try testing something out quickly tomorrow morning.

Reply:
Followup made here: #369